src/test/java/test/SetTest.java

```diff
@@ -0,0 +1,52 @@
+package test;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+    @BeforeEach
```

Comment:
인스턴스 변수와 생성자 혹은 메서드 구분을 위해 공백을 넣어주시면 가독성이 좋을 것 같아요 :)
```suggestion
    private Set<Integer> numbers;
    
    @BeforeEach
```

Reply:
리뷰 감사합니다^^